public-app-getting-started-template/README.md

```diff
@@ -18,6 +18,9 @@ NOTE: You need to be a super-admin in the developer account, where you will be u
 
 7. You can either update the default view or create a team view. Then add the UI Extension card to it.
 ![image](https://github.com/kemmerle/hubspot-public-app-template/assets/44659712/d8e74c43-f6d5-4f4e-8f52-f5324f55c565)
+8. For Webhooks, navigate to the objects that you have subscribed to in the CRM. For example, in this project only webhook subscriptions on the contact object were created, so navigate to the contacts tab in the CRM.
```

Comment:
@shambutennis, could you also add a step where the user needs to navigate to the `webhook.json` file to change the `targetUrl` setting? Could you also name the gate to which HubSpotters need to ungate their portals for webhooks? Thanks!